Manito/Manito/Screens/Letter/LetterViewController.swift

```diff
@@ -7,281 +7,198 @@
         Task {
             do {
-                let letterContent = try await self.letterSevice.fetchSendLetter(roomId: roomId)
-                
-                if let content = letterContent {
-                    dump(content)
-                    self.manitteeId = content.manittee?.id
-                    self.letterList = content.messages
+                let letterData = try await self.letterSevice.fetchSendLetter(roomId: roomId)
+                if let letterData {
+                    completionHandler(.success(letterData))
+                } else {
+                    completionHandler(.failure(.unknownError))
                 }
             } catch NetworkError.serverError {
-                print("serverError")
+                completionHandler(.failure(.serverError))
             } catch NetworkError.clientError(let message) {
-                print("clientError:\(String(describing: message))")
+                completionHandler(.failure(.clientError(message: message)))
             }
         }
     }
     
-    private func fetchReceviedLetter(roomId: String) {
+    private func fetchReceivedLetter(roomId: String, completionHandler: @escaping ((Result<Letter, NetworkError>) -> Void)) {
         Task {
             do {
-                let letterContent = try await self.letterSevice.fetchReceiveLetter(roomId: roomId)
-                
-                if let content = letterContent {
-                    dump(content)
-                    self.letterList = content.messages
+                let letterData = try await self.letterSevice.fetchReceiveLetter(roomId: roomId)
+                if let letterData {
+                    completionHandler(.success(letterData))
+                } else {
+                    completionHandler(.failure(.unknownError))
                 }
             } catch NetworkError.serverError {
-                print("serverError")
+                completionHandler(.failure(.serverError))
             } catch NetworkError.clientError(let message) {
-                print("clientError:\(String(describing: message))")
+                completionHandler(.failure(.clientError(message: message)))
             }
         }
     }
 }
 
+// MARK: - LetterViewDelegate
+extension LetterViewController: LetterViewDelegate {
+    func presentCreateLetterViewController() {
+        guard let manitteeId else { return }
+        let viewController = CreateLetterViewController(manitteeId: manitteeId,
+                                                        roomId: self.roomId,
+                                                        mission: self.mission,
+                                                        missionId: self.missionId)
+        let navigationController = UINavigationController(rootViewController: viewController)
+        viewController.configureDelegation(self)
+        self.present(navigationController, animated: true)
+    }
+
+    func fetchSendLetter() {
```

Comment:
p5;

```suggestion
    func fetchSendedLetter() {
```

아래의 함수가 fetchReceivedLetter() 이길래 위의 send도 sended가 되어야 하지 않나... 란 생각에 남깁니다..😊

Reply:
[93faf73](https://github.com/DeveloperAcademy-POSTECH/MC2-Team5-Firefighter/pull/413/commits/93faf73b7db9c75e914638be0e4783224d0029d0)

`fetchSentLetter`로 네이밍 변경했습니다!